tutorials/smart-contracts/deploy-erc20.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Deploy a ERC20 to Asset Hub
+description: Deploy an ERC20 token on Asset Hub using PolkaVM. This guide covers contract creation, compilation, deployment, and interaction via Polkadot Remix IDE.
+---
+
+# Deploy ERC20 to Asset Hub
+
+## Introduction
+
+[ERC20](https://eips.ethereum.org/EIPS/eip-20){target=\_blank} tokens are fungible tokens commonly used for creating cryptocurrencies, governance tokens, and staking mechanisms. Asset Hub enables easy token deployment with EVM-compatible smart contracts via PolkaVM. This tutorial covers deploying an ERC20 contract on the Westend TestNet using [Polkadot Remix IDE](https://remix.polkadot.io){target=\_blank}, a web-based development tool. [OpenZeppelin's ERC20 contracts](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/v5.0.0/contracts/token/ERC20){target=\_blank} are used for security and compliance.
+
+## Prerequisites
+
+Before starting, make sure you have:
+
+- [MetaMask](https://metamask.io/){target=\_blank} installed and connected to [Westend Asset Hub](https://chainlist.org/chain/420420421){target=\_blank}
+- A funded account with some WND tokens (you can get them from the [Westend Faucet](https://faucet.polkadot.io/westend?parachain=1000){target=\_blank})
+- Basic understanding of Solidity and fungible tokens
+
+## Creating the ERC20 Contract
+
+To create the ERC20 contract, you can follow the steps below:
+
+1. Navigate to the [Polkadot Remix IDE](https://remix.polkadot.io){target=\_blank}
+2. Click in the **Create new file** button under the **contracts** folder, and name your contract as `MyToken.sol`
+
+    ![](/images/tutorials/smart-contracts/deploy-erc20/deploy-erc20-1.webp)
+
+3. Now, paste the following ERC20 contract code into the editor
+
+    ```solidity title="MyToken.sol"
+    --8<-- 'code/tutorials/smart-contracts/deploy-erc20/MyToken.sol'
+    ```
+
+    The key components of the code above are:
+
+    - Contract Imports
+
+        - [`ERC20.sol`](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/token/ERC20/ERC20.sol){target=\_blank} - the base contract for fungible tokens, implementing core functionality like transfers, approvals, and balance tracking
```

Comment:
```suggestion
        - [**`ERC20.sol`**](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/v5.0.0/contracts/token/ERC20/ERC20.sol){target=\_blank} - the base contract for fungible tokens, implementing core functionality like transfers, approvals, and balance tracking
```

All list items in this format should have the text before (to the left of) the dash in bold, whether or not they contain a code element

Reply:
Please update all other instances